views/editprofile.view.php

```diff
@@ -136,7 +137,7 @@
                     <form method="post" onsubmit="return confirm('Are you sure you want to delete this hobby from your profile?');">
                         <input type="hidden" name="deleteHobbyUser" value="<?= $userHobby['hobby_name'] ?>">
                         <button type="submit" style="background: none; border: none; padding: 0; margin: 0;" name="delete_hobby_user">
-                            <hobbyarticle style="background-image: url('/views/public/images/hobby_<?= $userHobby['hobby_name'] ?>.jpg')">
+                            <hobbyarticle style="background-image: url('/views/public/images/hobby_<?= str_replace(' ', '_', $userHobby['hobby_name']) ?>.jpg')">
```

Comment:
dit had ik perongeluk verwijderd volgens mij dus die moet er wel in blijven voor de merge

Reply:
al gedaan :0